config/routes.rb

```diff
@@ -8,6 +8,6 @@
 
     get "translate", to: "translations#index"
     get "about", to: "static#about"
-    root to: "doctors#search"
+    root to: "doctors#search_root"
```

Comment:
Um, no I think that is a bad idea. Is there anyway to have page-specific JS on the root only @equivalentideas ??